packages/app/package.json

```
@@ -88,6 +88,7 @@
   },
   "dependencies": {
     "@cryptkeeperzk/providers": "workspace:^",
+    "@cryptkeeperzk/rln-proof": "workspace:0.1.0-beta.1",
```

Maybe it makes sense to use ^ version?

```suggestion
    "@cryptkeeperzk/rln-proof": "workspace:^",
```